kyo-prelude/jvm/src/test/scala/kyo2/MonadLawsTest.scala

```diff
@@ -1,5 +1,6 @@
 package kyo2
 
+import Flat.unsafe.bypass
```

Comment:
No need for a wrapper class anymore but we need to bypass the `Flat` check. It's safe since `zio-laws` can only compare two monad instances by calling the provided `Equal` and should never rely on the regular `==`, even if the monad instance is nested into another. In other words, the `A` provided to the `Equal` instance can not be nested.